package.json

```diff
@@ -47,8 +47,8 @@
     "async": "^0.9.0",
     "bluebird": "^2.9.14",
     "debug": "^2.1.3",
-    "elastic-deletebyquery": "1.0.7",
-    "elasticsearch": "^11.0.1",
+    "elastic-deletebyquery": "^1.0.8",
```

Comment:
@wolfgang-s - is an update from 1.0.7 to 1.0.8 required for working with 5.x? or if its for another reason, can it be part of a separate PR?